### DISCUSSION TITLE
Is it possible to do metasploit function after performing the following?

### DISCUSSION THREAD

**Discussion by flamecopper** [rapid7/metasploit-framework#16336] [Q&A]
ssh -D 9050 student@172.25.xxx.xxxx -p60000
/etc/proxychains.conf -> socks4 127.0.0.1 9050

I use nmap:
proxychains nmap 172.17.0.1
ProxyChains-3.1 (http://proxychains.sf.net)

Is it possible to do something like:
proxychains msfconsole?

Reason being, meterpreter is unstable.

**Comment by smcintyre-r7** [comment]
> Reason being, meterpreter is unstable.

It would be great if you could file a bug for that to help us address the root of the problem.
